app/views/green_lanes/results/_category_assessments_card.erb

```diff
@@ -7,25 +7,24 @@
 
   <div class="govuk-summary-card__content">
     <dl class="govuk-summary-list">
-      <% @assessments.public_send("cat_#{category}_exemptions_not_met").each do |category_assessment| %>
+      <% assessments = @cas_without_exemptions.present? ? @cas_without_exemptions : @assessments.public_send("cat_#{category}_exemptions_not_met") %>
```

Comment:
If there are any category assessments without exemptions then we show them and say `no exemptions available`